deploy.sh

```diff
@@ -89,35 +88,40 @@ idmsvc" \
 setup_debezium() {
   echo "Debezium is setting up.."
   download_debezium_configuration
-  cd scripts
-  chmod 0700 kafka.sh
-  ./kafka.sh -n `oc project -q`
-  cd ..
-
-  RBAC_KAFKA_POD=rbac-kafka-kafka-0
-  until oc get pod $RBAC_KAFKA_POD  > /dev/null 2>&1; do
-    echo "Waiting for pod ${RBAC_KAFKA_POD} to be created..."
-    sleep 5
-  done
 
-  oc wait pod $RBAC_KAFKA_POD --for=condition=Ready --timeout=60s
+  NAMESPACE=env-$(oc project -q)
+  oc process -f ./deploy/debezium-connector.yml -p KAFKA_CONNECT_INSTANCE="$NAMESPACE" --param-file=./scripts/connector-params.env | oc apply -f -
 
-  oc rsh $RBAC_KAFKA_POD /opt/kafka/bin/kafka-topics.sh \
-  --bootstrap-server=rbac-kafka-kafka-bootstrap:9092 \
+  # TODO: remove the following lines with topics set directly in the rbac ClowdApp template, as in
+  # https://github.com/tonytheleg/inventory-api/blob/8db7dbbaca054193c19d2cc109fe152a24e51e29/deploy/kessel-inventory-ephem.yaml#L82
+  KAFKA_POD=$(oc get pod/"$NAMESPACE"-kafka-0 --no-headers -o custom-columns=":metadata.name")
+  oc wait pod "$KAFKA_POD" --for=condition=Ready --timeout=60s
+  KAFKA_BOOTSTRAP=$(oc get svc/"$NAMESPACE"-kafka-bootstrap --no-headers -o custom-columns=":metadata.name")
+  oc rsh $KAFKA_POD /opt/kafka/bin/kafka-topics.sh \
```

Comment:
Not a blocker or anything but maybe redundant to get the pod name and service name when you know them already. Not sure what two less oc get calls does for speed/performance but figured id point it out

```shell
oc rsh ${NAMESPACE}-kafka-0 /opt/kafka/bin/kafka-topics.sh \
  --bootstrap-server=${NAMESPACE}-kafka-bootstrap:9092 \
  --create --if-not-exists --topic outbox.event.workspace --partitions 3 --replication-factor 1
```

There is also jsonpath if you still want to call get specifically and haven't seen that before
`KAFKA_POD=$(oc get pod/"$NAMESPACE"-kafka-0 -o jsonpath='{.metadata.name})`